packages/jazz-vue/src/createJazzVueApp.ts

```diff
@@ -244,7 +251,7 @@ export function createJazzVueApp<Acc extends Account>({
 
   function useCoState<V extends CoValue, D>(
     Schema: CoValueClass<V>,
-    id: ID<V> | undefined,
+    id: ComputedRef<ID<V> | undefined>,
```

Comment:
Is this the only way to make it reactive to `id`? I don't know enough about Vue but the call site looks very elaborate...

Reply:
A plain `id` reference is not tracked and will not reactively update useCoState(). 

In this example folders will be undefined:
```
const computedFoldersId = me.value?.root?.folders?.id
const folders = useCoState(FolderList, computedFoldersId, [{ items: [{}] }]);
```

In this example we track changes to `me.value?.root?.folders?.id` and `useCoState()` tracks changes to computedFoldersId: 
```
const computedFoldersId = computed(() => me.value?.root?.folders?.id);
const folders = useCoState(FolderList, computedFoldersId, [{ items: [{}] }]);
```

Reply:
Now `id` is `MaybeRef<ID<V> | undefined>`

It accepts either a plain value, or a ref.